cibyl/sources/zuul/zuul_utils.py

```diff
@@ -0,0 +1,18 @@
+import yaml
+
+
+def encrypted_constructor(loader: yaml.SafeLoader,
+                          node: yaml.nodes.MappingNode):
+    """
+    Construct an ecrypted token.
+    """
+    return ""
```

Comment:
Is this meant to be left empty?

Reply:
Yes. Since we are not replacing `!encrypted/pkcs1-oaep` with anyting.